### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by CaptainMorganCodes**
* This has comments in app.js file
* docs are hosted at /docs

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+ * @constant PORT
+ * @type {number} 
+ * port number
+ */
 const PORT = process.env.PORT || 3000;
-const server = express()
-  .listen(PORT, () => console.log(`Listening on ${PORT}`));
+const app = express();
 
-const io = socketIO(server);
```
Comment (mts513) [inline server/app.js]: I had to work with an instructor to fix app & express to host the docs

**Review by CaptainMorganCodes — COMMENTED** [review]
This is still largely incomplete but I wanted to get working hosted docs. This was actually really helpful because I did run into some roadblocks. All in all, its trucking along =)

File: server/app.js [hunk 2]
```diff
@@ -1,15 +1,30 @@
...
```
Comment (CaptainMorganCodes) [inline server/app.js]: I had to work with an instructor to fix app & express to host the docs

File: README.md [hunk 3]
```diff
@@ -10,7 +10,7 @@
 ## Web Application
 ***[Explain your app, should be at least a paragraph. What does it do? Why should I use? Sell your product!]***
 
-The application is a communication platform for realtime chat between two users. We will be utilizing NodeJS, JavaScript, Google Translate API, and AWS. 
+The application is a communication platform for realtime chat between two or more users. We will be utilizing NodeJS, JavaScript, Google Translate API, and AWS.
```
Comment (Martinbalke) [inline README.md:13]: We're not using AWS anymore

Comment (Martinbalke) [inline README.md:13]:
```suggestion
The application is a communication platform for realtime chat between two or more users. We will be utilizing NodeJS, JavaScript, Google Translate API.
```

**Review by Martinbalke — APPROVED** [review]

**Pull request merged**